src/PlaygroundWithGit.jl

```diff
@@ -44,9 +44,12 @@ function loan_interest_calculator()
     end
     x = (a + b) / 2
     fx = f(x)
+    if abs(x) < 1e-6 || b - a < 1e-12
```

Comment:
I just noticed that `abs(x)` is not correct. You can remove this, since it is already on the while condition.